test/renderers/render.test.ts

```diff
@@ -53,24 +53,39 @@ describe("render()", () => {
     ]
 
     expect(await render(contentTypes, locales)).toMatchInlineSnapshot(`
-"// THIS FILE IS AUTOMATICALLY GENERATED. DO NOT MODIFY IT.
+      "// THIS FILE IS AUTOMATICALLY GENERATED. DO NOT MODIFY IT.
 
-import { Asset, Entry } from \\"contentful\\"
-import { Document } from \\"@contentful/rich-text-types\\"
+      import { Asset, Entry } from \\"contentful\\"
+      import { Document } from \\"@contentful/rich-text-types\\"
 
-export interface IMyContentTypeFields {
-  /** Array field */
-  arrayField: (\\"one\\" | \\"of\\" | \\"the\\" | \\"above\\")[]
-}
+      export interface IMyContentTypeFields {
+        /** Array field */
+        arrayField: (\\"one\\" | \\"of\\" | \\"the\\" | \\"above\\")[]
+      }
 
-export interface IMyContentType extends Entry<IMyContentTypeFields> {}
+      export interface IMyContentType extends Entry<IMyContentTypeFields> {
+        sys: {
+          id: string
+          type: string
+          createdAt: string
+          updatedAt: string
+          locale: string
+          contentType: {
+            sys: {
+              id: \\"myContentType\\"
+              linkType: \\"ContentType\\"
+              type: \\"Link\\"
+            }
+          }
+        }
+      }
 
-type CONTENT_TYPE = \\"myContentType\\"
+      type CONTENT_TYPE = \\"myContentType\\"
 
-type LOCALE_CODE = \\"en-US\\" | \\"pt-BR\\"
+      type LOCALE_CODE = \\"en-US\\" | \\"pt-BR\\"
 
-type CONTENTFUL_DEFAULT_LOCALE_CODE = \\"en-US\\"
-"
-`)
+      type CONTENTFUL_DEFAULT_LOCALE_CODE = \\"en-US\\"
+      "
+    `)
```

Comment:
Not sure what caused jest to start rendering nicely-indented inline snapshots, but I'm a big fan.